.github/workflows/inferrer-python-ci.yml

```diff
@@ -0,0 +1,38 @@
+name: "Inferrers: Run checks and build"
+on: [ push ]
+# paths: [ 'pipeline/inferrer/**' ]
```

Comment:
Does this need adding back in, post testing?

Reply:
Whoops, yes.